client/src/features/Lottery/CustomDesign.tsx

```diff
@@ -1,9 +1,16 @@
+import { motion } from "framer-motion";
+import { DISSOLVE, SCROLL_MOTION } from "@/constants/animation.ts";
+import { SectionKey } from "@/types/scrollHeaderStyle.ts";
 import Description from "./Description";
 import Section from "./Section";
 
-export default function CustomDesign() {
+interface CustomDesignProps {
+    id: SectionKey;
+}
```

Comment:
이 id props 부분은 스크롤 애니메이션이 필요한 섹션에 공통적으로 들어가는 속성이니까 공통 interface로 빼는건 어떨까용

```suggestion
// types/[...].ts
export interface SectionProps {
    id: SectionKey;
}

export default function CustomDesign({ id }: SectionProps) {
  ...
}

// 만약에 다른 props가 있다면

interface CustomDesignProps extends SectionProps {
    ...
}

export default function CustomDesign({ id, ...props }: CustomDesignProps) {
  ...
}
```

Reply:
반영하겠슴다!! 친절한 설명 감사합니다 ㅎㅎ